clients/admin-ui/src/features/common/table/v2/FidesTable.tsx

```diff
@@ -26,7 +26,8 @@ import React, { ReactNode, useMemo, useState } from "react";
 
 import { DisplayAllIcon, GroupedIcon } from "~/features/common/Icon";
 import { FidesRow } from "~/features/common/table/v2/FidesRow";
-import { getTableTHandTDStyles } from "~/features/common/table/v2/util";
+
+import { getTableTHandTDStyles } from "./util";
```

Comment:
```suggestion
import { getTableTHandTDStyles } from "~/features/common/table/v2/util";
```

Reply:
Is this a preferred pattern, to have the more full path? VS Code leans toward shortest, so I will keep an eye on that.

Reply:
yes we prefer the absolute path